interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/PublisherToStreamTest.scala

```diff
@@ -0,0 +1,42 @@
+package scalaz.zio.interop.reactiveStreams
+
+import org.scalatest.{ Matchers, WordSpec }
+import scalaz.zio.Exit.Cause
+import scalaz.zio.stream.{ Sink, Stream }
+import scalaz.zio.{ DefaultRuntime, Exit }
+
+class PublisherToStreamTest extends WordSpec with Matchers with DefaultRuntime {
+
+  "A PublisherStream" should {
+
+    def publish(inputData: Stream[Any, Throwable, Int]): Exit[Throwable, List[Int]] = {
+      val resultZIO =
+        for {
+          publisher <- inputData.toPublisher
```

Comment:
Could it be useful to test these conversions using an Akka Streams source?

Reply:
done